manifest/manifest_test.go

```diff
@@ -27,19 +27,19 @@ var base = manifest.Manifest{
 			PubKey: gpbft.PubKey{1},
 		},
 	},
-	GpbftConfig: manifest.GpbftConfig{
+	Gpbft: manifest.GpbftConfig{
 		Delta:                10,
 		DeltaBackOffExponent: 0.2,
 		MaxLookaheadRounds:   10,
 	},
-	EcConfig: manifest.EcConfig{
-		ECFinality:               900,
+	EC: manifest.EcConfig{
+		Finality:                 900,
 		CommitteeLookback:        5,
```

Comment:
Move this into the GPBFT section?

Reply:
Yes

Reply:
No

Reply:
Hmm, gpbft section gets passed to GPBFT and GPBFT does not care about committee lookback, the lookback is external to it.

Reply:
I moved it to the root.